browser/src/react-redux/App/Login/Login.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+
+export default class Login extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isValidated: false,
+      validEmail: false,
+      validPassword: false,
+      email: '',
+      password: '',
+      emailError: '',
+      passwordError: ''
+    }
+    this.handleInputChange = this.handleInputChange.bind(this)
```

Comment:
Just use arrow functions below instead of manually binding the functions up here. Less noise.